UnitTests/test_saveFile.py

```diff
@@ -15,17 +15,21 @@
 
 def test_SaveFile():
 
-    if (os.path.isfile("/testResults/save.rs9")):
-        os.remove("/testResults/save.rs9")
+    savePath = r"\testResults\save.rs9"
```

Comment:
savePath variable is kinda redundant since it is used only once. The bellow looks better to me. Also the string should have forward slashes not backward. 
filepath = dirName + "/testResults/save.rs9"